pkg/maps/tunnel/tunnel.go

```diff
@@ -60,7 +60,7 @@ func NewTunnelMap(mapName string) *Map {
 		MaxEntries,
 		0,
 		bpf.ConvertKeyValue,
-	).WithCache().WithPressureMetric().WithNonPersistent().
+	).WithCache().WithPressureMetric().
```

Comment:
So where is the matching `Recreate` call?

Reply:
See `tunnel.TunnelMap().Recreate()` and `vtep.VtepMap().Recreate()` in `daemon/cmd/datapath.go` for both 'NonPersistent' maps.